Resolvers/CommandLineResolver.py

```diff
@@ -56,7 +56,7 @@ def addPlatform(self, datastore, platformName):
             while not nationalityCheckOk:
                 newNationalityInput = input("Please type name of new nationality: ")
                 nationalityCheckOk = datastore.checkNationality(newNationalityInput)
-            chosenNationality = datastore.addNationality(newNationalityInput)
+            chosenNationality = datastore.addNationalities(newNationalityInput)
```

Comment:
we are just adding a singular nationality here. The singular method name seemed to make more sense.

Reply:
Add methods renamed.

https://github.com/debrief/python-import-library/pull/15/commits/29b87d2cc9d16d11caa51380761e8739f57550e1